calculator.rb

```diff
@@ -0,0 +1,139 @@
+# ROOM FOR IMPROVEMENT
+# Support parentheticals
+# Support commas in longer numbers
+# Allow for equation with more than 2 numbers
+# Support square root
+# Ask whether to end or continue after outputting result
+
+# -------------
+
+# CHECKS IF USER INPUT IS A VALID OPERATION
+def is_operator?(user_input)
```

Comment:
So names are an interesting thing. Usually when you use a `is` prefix, or add `?` to the end of a function name, it signals that the return value will be a boolean. A potentially better name for this function could be `get_operator`